docs/csi-migrated-volume-support.md

```diff
@@ -0,0 +1,48 @@
+# CSI migrated volume support
+
+Velero plugin for vsphere v1.4.0 and above supports backing up of migrated vsphere csi volumes. A migrated vsphere csi volume is original provisioned by vmware vsphere cloud provider but migrated to vsphere csi driver. 
```

Comment:
Velero Plugin for vSphere

General comment: vsphere -> vSphere
vmware -> VMware
csi -> CSI